website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

```diff
@@ -298,13 +298,16 @@ If you believe your project might be affected, read more details in the migratio
-
-Once you have your project compiling and running on the latest version of dbt in the development environment for your `dbt-version-upgrade` branch, try replicating one of your production jobs to run off your branch's code. You can do this by creating a new deployment environment for testing, setting the custom branch to 'ON' and referencing your `dbt-version-upgrade` branch. You'll also need to set the dbt version in this environment to the latest dbt Core version.
-
-
-
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions/upgrade-environment.png" title="Setting your testing environment" />
-
-
-Then add a job to the new testing environment that replicates one of the production jobs your team relies on. If that job runs smoothly, you should be all set to merge your branch into main and change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.
+Once you know what code changes you'll need to make, you can start implementing them. We recommend you:
+- Create a separate dbt project, "Upgrade project", to test your changes before making them live in your main dbt project.
+- In your "Upgrade project", connect to the same repository you use for your production project.
+- Set the development environment [settings](/docs/dbt-versions/upgrade-dbt-version-in-cloud) to run the latest version of dbt Core.
+- Check out a branch `dbt-version-upgrade`, make the appropriate updates to your project, and verify your dbt project compiles and runs with the new version in the dbt Cloud IDE.
+  - If upgrading directly to the latest version results in too many issues, try testing your project iteratively on successive minor versions. There are years of development and a few breaking changes between distant versions of dbt Core (for example, 0.14 --> 1.0). The likelihood of experiencing problems upgrading between successive minor versions is much lower, which is why upgrading regularly is recommended.
+- Once you have your project compiling and running on the latest version of dbt in the development environment for your `dbt-version-upgrade` branch, try replicating one of your production jobs to run off your branch's code.
+- You can do this by creating a new deployment environment for testing, setting the custom branch to 'ON' and referencing your `dbt-version-upgrade` branch. You'll also need to set the dbt version in this environment to the latest dbt Core version.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions/upgrade-environment.png" width="80%" title="Setting your testing environment" />
+
+- Then add a job to the new testing environment that replicates one of the production jobs your team relies on.
+  - If that job runs smoothly, you should be all set to merge your branch into main and, change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.
```

Comment:
@nataliefiann , i turned it into two separate bullets since. i wasn't sure if the `and,` was a right fit.

````suggestion
  - If that job runs smoothly, you should be all set to merge your branch into main. 
  - Then change your development and deployment environments in your main dbt project to run off the newest version of dbt Core.
```